<launch>
    <node pkg="imcoder_reader" name="imcoder_reader" type="imcoder_reader_node" output="screen">
        <param name="calibration_file_path"             value="$(find imcoder_reader)/config"/>
        <param name="calibration_file_name"             value="RTIMULib.ini" />
        <param name="frame_id"                          value="imu"/>
        <param name="imu_topic_name"                    value="imu/data"/>
        <param name="mag_topic_name"                    value="imu/mag"/>
        <param name="update_rate"                       value="10"/>
    </node>
</launch>
